src/scripts/helpers/zotonic_setup

```diff
@@ -78,6 +78,8 @@ function find_config {
 export -f find_config 
 
 function find_config_arg {
+    [ ! -z "$ZOTONIC_CONFIG_DIR" ] && { echo "-config ${ZOTONIC_CONFIG_DIR}/$1"; return 0; }
+
```

Comment:
Why is this needed? The `$ZOTONIC_CONFIG_DIR` check is already in `find_config`, which is called directly below.